client/signup/difm/components/BrowserView.tsx

```diff
@@ -3,13 +3,13 @@ import {
 	ABOUT_PAGE,
 	BLOG_PAGE,
 	CONTACT_PAGE,
+	CUSTOM_PAGE,
 	FAQ_PAGE,
 	HOME_PAGE,
 	PHOTO_GALLERY_PAGE,
 	PRICING_PAGE,
-	SERVICE_SHOWCASE_PAGE,
+	type PageId,
 	SHOP_PAGE,
-	SITEMAP_PAGE,
```

Comment:
These have been deprecated and are no longer in use:

https://github.com/Automattic/wp-calypso/blob/3d91f1ca979992cb45d329b26899be635a2bcc05/client/signup/difm/constants.tsx#L36-L41